executor/batch_point_get.go

```diff
@@ -160,3 +161,42 @@ func (e *BatchPointGetExec) initialize(ctx context.Context) error {
 	e.handles = handles
 	return nil
 }
+
+func (e *BatchPointGetExec) batchGet(ctx context.Context, keys []kv.Key) (res map[string][]byte, err error) {
```

Comment:
Why add a new batchGet instead of using https://github.com/pingcap/tidb/blob/fbf0d90c06f19a69304d430950be60cbfc257765/session/txn.go#L229 ?

Reply:
I copy the code from PointGetExec and forget this one.
Fixed now.